composer.json

```diff
@@ -78,7 +82,13 @@
     "post-create-project-cmd": [
       "@cleanup",
       "WordPressProject\\composer\\ScriptHandler::createRequiredFiles"
+    ],
+    "code-sniff": [
+      "./vendor/bin/phpcs --config-set installed_paths ./vendor/wp-coding-standards/wpcs",
+      "./vendor/bin/phpcs ./web/wp-content"
+    ],
+    "unit-test": [
+      "./vendor/bin/phpunit tests/unit/*"
```

Comment:
> How can we exclude plugins/themes from our tests? Run a test for each plugin folder individually?

Currently I have the `install_dev_dependencies` job in Circle which installs all Composer dependencies (dev and require). The `vendor` and `web` directories are then saved to a persistent workspace so they can be used by other jobs.

The `code_sniff_unit_test` only restores `vendor` leaving `web` as it is in GitHub, only containing custom code. Unit tests and code sniffing are then run against the custom code only.